reassembler.py

```diff
@@ -1457,7 +1445,7 @@ def parity7(data):
         if station:
             m['source']['station_id'] = station
 
-        if jsonout:
+        if args['json']:
```

Comment:
Did you test this? I think this won't work this way.

Reply:
Fixed.